data/ga/legislature/Scott-Hilton-91abe321-5ea4-4f17-b8d8-573d7cb913e6.yml

```diff
@@ -0,0 +1,32 @@
+id: ocd-person/91abe321-5ea4-4f17-b8d8-573d7cb913e6
+name: Scott Hilton
+given_name: Scott
+family_name: Hilton
+email: scott.hilton@house.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/hilton-scott-4899.jpg
+party:
+- name: Republican
+roles:
+- type: lower
```

Comment:
@sroomf this person was in house district 95 between January 2017 and january 2019. They may have another file elsewhere.